packages/editor/src/components/post-saved-state/index.js

```diff
@@ -128,45 +129,57 @@ export default function PostSavedState( {
 		text = shortLabel;
 	}
 
+	const buttonAccessibleLabel = text || label;
+
+	/**
+	 * The tooltip needs to be enabled only is the button is not disabled. When
+	 * relying on the internal Button tooltip functionality, this causes the
+	 * resulting `button` element to be always remove and re-added to the DOM,
```

Comment:
Spotted a couplle typos: `only is the button` should be `only if the button`, ` always remove and` should be `always removed and`.

Reply:
https://github.com/WordPress/gutenberg/pull/56502/commits/7bce2ea963ac5c37258ae7f5995041905077afba